src/PrestaShopBundle/Form/Admin/Sell/Customer/PrivateNoteType.php

```diff
@@ -44,6 +60,9 @@ public function buildForm(FormBuilderInterface $builder, array $options)
             ->add('note', TextareaType::class, [
                 'required' => false,
                 'empty_data' => '',
+                'attr' => [
+                    'placeholder' => $this->translator->trans('Add a note to the customer account. This note is only visible to you.', [], 'Admin.Orderscustomers.Feature'),
```

Comment:
```suggestion
                    'placeholder' => $this->translator->trans('Add a note on this customer. It will only be visible to you.', [], 'Admin.Orderscustomers.Feature'),
```